contracts/Core.sol

```diff
@@ -22,35 +22,49 @@ pragma solidity ^0.4.23;
 // ----------------------------------------------------------------------------
 
 import "./CoreInterface.sol";
+import "./MerklePatriciaProof.sol";
+import "./util.sol";
+import "./WorkersInterface.sol";
+import "./RLP.sol";
 
 
 /// @dev Core is a minimal stub that will become the anchoring and consensus point for
 ///      the utility chain to validate itself against
-contract Core is CoreInterface {
+contract Core is CoreInterface, Util {
```

Comment:
Yes because util is not a library contract. In the source eth-patricia repo also, it was not a library contract. It's needed because we are using below function:
bytes32ToBytes

If we don't inherit compiling gives below error:
 DeclarationError: Undeclared identifier.

What do you suggest?